services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py

```diff
@@ -106,7 +106,6 @@ async def get_containers_activity(
 
     container_name = inactivity_command.service
 
-    inactivity_response: str | None = None
     try:
         inactivity_response = await run_command_in_container(
```

Comment:
what happens if inactivity_response is None?

Reply:
The `inactivity_response` is always a string when this succeeds.
Also below code will also handle the `None`, which will never happen.